Troubleshooting.md

```diff
@@ -0,0 +1,77 @@
+# Troubleshooting
+
+This document will help troubleshoot the common debugging errors.
+
+## Java Language Support extension is not started or broken.

```

Comment:
`or broken` is hard to understand, 'is not started' => 'fails to start'

Reply:
fixed.